client/src/components/FeatureCard/FeatureCardList.tsx

```diff
@@ -0,0 +1,13 @@
+import FeatureCardItem from './FeatureCardItem';
+
+const FeatureCardList = ({ recipes }) => {
+  return (
+    <ul className='flex flex-col items-stretch'>
+      {recipes.map((recipe) => (
+        <FeatureCardItem recipe={recipe} />
```

Comment:
```suggestion
        <FeatureCardItem {...recipe} />
```
Truyền như thế này thì lúc pass props cho component k cần phải như này nữa
```
const FeatureCardItem = ({ recipe }) => {
  const { image, title, authorImage, author, likes, reviews } = recipe;
```

Mà chỉ cần
`const FeatureCardItem = ({ image, title, authorImage, author, likes, reviews }) => {`


Tương tự với component CategoryItem